src/traces/pie/attributes.js

```diff
@@ -6,6 +6,7 @@
 * LICENSE file in the root directory of this source tree.
 */
 
+'use strict';
```

Comment:
there's no functions in this, so no need for `'use strict';`

That said, I don't mind making `'use strict'` for all files even to one without functions.